src/main/java/com/vmware/vipclient/i18n/base/cache/CacheItem.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vipclient.i18n.base.cache;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CacheItem {
+	public CacheItem() {
+		
+	}
+	
+	public CacheItem (Map<String, String> dataMap, String etag, long timestamp, Long maxAgeMillis) {
+		super();
+		if (dataMap != null) 
+			this.addCachedData(dataMap);
+		this.etag = etag;
+		this.timestamp = timestamp;
+		this.maxAgeMillis = maxAgeMillis;
+	}
+	
+	public CacheItem (Map<String, String> dataMap) {
+		super();
+		if (dataMap != null)
+			this.addCachedData(dataMap);
+	}
+	
+	private final Map<String, String> cachedData = new HashMap<String, String>();
+	private String etag;
+	private long timestamp;
+	private Long maxAgeMillis;
```

Comment:
should move maxAgeMillis to MessageCache because the value is same for different CacheItem instance.